cmd/scollector/collectors/exim_linux.go

```diff
@@ -0,0 +1,113 @@
+package collectors
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"bufio"
```

Comment:
Will fix... Go imports failed me, or I failed it
On Jan 15, 2015 1:01 PM, "Matt Jibson" <notifications@github.com> wrote:

> In cmd/scollector/collectors/exim_linux.go
> <https://github.com/bosun-monitor/bosun/pull/658#discussion-diff-23029016>
> :
>
> > @@ -0,0 +1,113 @@
> > +package collectors
> > +
> > +import (
> > +	"fmt"
> > +	"os/exec"
> > +	"strconv"
> > +	"strings"
> > +	"time"
> > +
> > +	"bufio"
>
> Why are these separate?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bosun-monitor/bosun/pull/658/files#r23029016>.
>